processors/tenantidprocessor/testdata/config.yml

```diff
@@ -0,0 +1,16 @@
+receivers:
+  examplereceiver:
+
+processors:
+  hypertrace_tenantid:
+    tenantid_header_name: header-tenant
+    tenantid_attribute_key: attribute-tenant
```

Comment:
the tenantid prefix seems redundant I would remove it.

Reply:
The indeed feel redundant but I am curious about the need to be configurable at least for header_name cc @davexroth

Reply:
the config is optional, if no provided it uses the default that the platform uses

Reply:
agreed, the tenantid_ prefix is redundant, feel free to remove.

having the key configurable is useful, as we've configured it with different headers in different deployments.